src/Features/Core/Portable/RemoveUnusedParametersAndValues/AbstractRemoveUnusedValuesCodeFixProvider.cs

```diff
@@ -594,6 +620,13 @@ void InsertLocalDeclarationStatement(TLocalDeclarationStatementSyntax declaratio
                 }
                 else if (insertionNode is TStatementSyntax)
                 {
+                    // If the insertion node is being removed, keep the leading trivia with the new declaration.
+                    if (nodesToRemove.TryGetValue(insertionNode, out var isProcessed) && !isProcessed)
+                    {
+                        declarationStatement = declarationStatement.WithLeadingTrivia(insertionNode.GetLeadingTrivia());
+                        // Mark the node as processed so that the trivia only gets added once.
+                        nodesToRemove[insertionNode] = true;
```

Comment:
i'd much rather just have a second set that indicates if the node was processed or not.  then there's no need for this less clear map of nodes to booleans :)

Reply:
Works for me, I'll do it that way.